### PR TITLE
update documentation for pagerduty_extension

### LINKED ISSUE #131: Error [Url can't be empty] with resource "pagerduty_extension"
### Terraform Version
Terraform v0.11.13

### Affected Resource(s)
pagerduty_extension

### Terraform Configuration Files
```hcl
resource "pagerduty_extension" "slack"{
  name = "My Web App Extension"
   extension_schema = "${data.pagerduty_extension_schema.webhook.id}"
  extension_objects    = ["${pagerduty_service.example.id}"]

  config = <<EOF
{
    "restrict": "any",
    "notify_types": {
            "resolve": false,
            "acknowledge": false,
            "assignments": false
    },
    "access_token": "XXX"
}
EOF

}.
```

Here is the full code:
https://github.com/heldersepu/hs-scripts/blob/master/TerraForm/pagerduty/pagerduty.tf

### Debug Output
```
Error: Error applying plan:

1 error(s) occurred:

* ....pagerduty_extension.slack: 1 error(s) occurred:

* pagerduty_extension.slack: 
  POST API call to https://api.pagerduty.com/extensions failed 400 Bad Request. 
  Code: 2001, Errors: [Url can't be empty], Message: Invalid Input Provided
```

### Expected Behavior
It should complete without errors?

### Actual Behavior
We get that strange 'Url can't be empty' error but according to the documentation the endpoint_url is optional

### PR DESCRIPTION
**Pull request by karancode**
Resolves #131

### CODE REVIEW

File: website/docs/r/extension.html.markdown [hunk 1]
```diff
@@ -71,7 +71,8 @@ EOF
 The following arguments are supported:
 
   * `name` - (Optional) The name of the service extension.
-  * `endpoint_url` - (Optional) The url of the extension.
+  * `endpoint_url` - (Reqeuired|Optional) The url of the extension.
```
Comment (stmcallister) [inline website/docs/r/extension.html.markdown]: Can you fix the spelling on Required here?

Comment (pdecat) [reply website/docs/r/extension.html.markdown]: What about leaving it `Optional` as that's how it is defined on the terraform schema, so terraform does not enforce it, but being more explicit about some exceptions for `Generic V2 Webhook` like described in https://github.com/terraform-providers/terraform-provider-pagerduty/issues/131#issuecomment-489375555 ?

Comment (karancode) [reply website/docs/r/extension.html.markdown]: 22cd0e6602247e99626af8afd405d3e10f60c018 fixed the typo @stmcallister

Comment (stmcallister) [reply website/docs/r/extension.html.markdown]: I like the additional note! Nice job!

**Pull request merged**